Assets/Scripts/Scenes/InGame/Enemy/Trace/Feature/FeatureView.cs

```diff
@@ -13,17 +16,51 @@ public class FeatureView : MonoBehaviour
         private GameObject[] _stageInteracts;
         private GameObject[] nearStageObject = null;
         private GameObject interactTarget = null;
+        private GameObject _floor;
         AudioSource _audioSource;
         private Subject<Unit> _onDestroy = new Subject<Unit>();
         public IObservable<Unit> onDestroy { get => _onDestroy; }
+        public ReactiveProperty<GameObject> TemperatureTrace = new ReactiveProperty<GameObject>();
+        private Vector3 direction = new Vector3(0, -1, 0);
+        public StageTile stagetile;
```

Comment:
```suggestion
        private StageTile _stagetile;
        public StageTile stagetile { get { return _stagetile; } }
```
値を直接他のクラスから触れるのはまずいので、カプセル化してほしいです